src/main/java/org/springframework/data/tarantool/core/convert/DateConverters.java

```diff
@@ -26,7 +26,12 @@ public class DateConverters {
         List<Converter<?, ?>> converters = new ArrayList<Converter<?, ?>>();
 
         boolean useISOStringConverterForDate = Boolean
-                .parseBoolean(System.getProperty("org.springframework.data.couchbase.useISOStringConverterForDate", "false"));
+                .parseBoolean(
+                        System.getProperty(
+                                "org.springframework.data.couchbase.useISOStringConverterForDate",
```

Comment:
@Elishtar Couchbase! 😄 (heh, but now it's easier to remember where I picked this up)

Reply:
@akudiyar Classic))